src/FlipMove.js

```diff
@@ -770,4 +770,6 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
   }
 }
 
-export default propConverter(FlipMove);
+const enhancedFlipMove = /* #__PURE__ */ propConverter(FlipMove);
+
+export default enhancedFlipMove;
```

Comment:
Curious about the changes to this file. What's `/* #__PURE__ */`?

Reply:
i think its an uglify magic comment. means it can strip out the function if `enhancedFlipMove` is not used anywhere. https://davidwalsh.name/compress-uglify (search for pure_func )

Reply:
Exactly, it means that this can be safely dropped if its result is not used anywhere. It doesnt matter for the library itself (as we are sure that this result is used within the library), might slightly matter if this library gets used in application or other library.

By potentially dropping (but ofc only if `enhancedFlipMove` is not used) this call (`propConverter(FlipMove)`) it will allow the minifier to drop the whole lib (`FlipMove` etc) as it wont be referenced anymore.

Reply:
ohhh, awesome! Thanks for the explanation.